ui/pages/settings/experimental-tab/experimental-tab.component.js

```diff
@@ -109,6 +111,37 @@ export default class ExperimentalTab extends PureComponent {
     );
   }
 
+  renderToggleRedesignedConfirmations() {
+    const { t } = this.context;
+    const { redesignedConfirmations, setRedesignedConfirmationsEnabled } =
+      this.props;
+
+    return (
+      <Box
+        ref={this.settingsRefs[0]}
```

Comment:
Can we add this toggle to the settings search?

Reply:
I updated the PR to address this.